html/index.html

```diff
@@ -0,0 +1,50 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>user form</title>
+    </head>
+    <body>
+        <div>
+            <form method="post" action="#">
+                <h2>REGISTRATION FORM</h2>
+                <label for="Name">First Name</label>
+                <input type="name" name="name" id="Name"> <br>
+                <label for="Lastname">Lastname</label>
+                <input type="text" name="lastname" id="Lastname"> <br>
+                <label for="Email">Email</label>
+                <input type="email" id="Email" name="Email"> <br>
+                <label for="phone">Phone number</label>
+                <input type="tel" name="phonemumber" id="phone"> <br>
+                <label for="address">Address</label>
+                <input type="text" name="address" id="address"> <br>
+                <label for="username">Username</label>
+                <input type="text" name="username" id="username"> <br>
+                <label for="password">Password</label>
+                <input type="password" name="password" id="password"> <br>
+                <label>Gender</label>
+                <label for="Male">Male</label>
+                <input type="radio" name="male" id="Male" value="male" required>
```

Comment:
When using radio buttons, the user should be able to make only one selection, but with yours, both options can be selected. Kindly identify the problem and fix it.